packages/cli/package.json

```diff
@@ -34,7 +34,8 @@
   },
   "bin": {
     "hs": "./bin/hs",
-    "hscms": "./bin/hscms"
+    "hscms": "./bin/hscms",
+    "hubspot": "./bin/hubspot"
```

Comment:
do we also need to create this file in https://github.com/HubSpot/hubspot-cli/tree/master/packages/cli/bin?